src/main/telemetry/frsky.c

```diff
@@ -190,7 +191,14 @@ static void sendThrottleOrBatterySizeAsRpm(void)
 {
     sendDataHead(ID_RPM);
     if (ARMING_FLAG(ARMED)) {
-        serialize16(rcCommand[THROTTLE] / BLADE_NUMBER_DIVIDER);
+        if(rcCommand[THROTTLE] <= escAndServoConfig->minthrottle){
```

Comment:
The test should check rcData against rxConfig->mincheck. 

minthrottle/mincommand/rcCommand[] should be probably sent to receiver (that is value actually used in mixer/motors). But this depends on what you want on telemetry ...